src/components/shared/socialAuth.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import google from '../../logo/iconfinder_Google_1298745.png';
+import fb from '../../logo/iconfinder_square-facebook_317727.png';
+
+const socialAuth = () => {
+  const baseUrl = 'https://mervels-bn-backend-staging.herokuapp.com';
```

Comment:
I think this can be imported from `/src/config`

Reply:
okay